src/pages/configeditor/download_docker.md

```diff
@@ -158,6 +158,9 @@ You can test what you use here by running:<br/>
 Back up your properties file.  Everytime you update it, the file will be overwritten and you'll want to merge back any changes you made here.<br/>
   `cp acd-ce.properties acd-ce.properties.bak`
 
+Sign on to the container regisry with the credentails provided when purchasing the product.<br/>
+  `Docker login <container-registry> -u <application-ID> -p <password>`
```

Comment:
typo - should be "credentials" here

Reply:
Do you want to leave <container-registry> here, or be more specific and use "acdcontainerregistry.azurecr.io"? With username updated to <application-ID>, that's specific to Azure. Do we mention any where else in the config editor documentation what registry is being used for the product?